### PR TITLE
[FIX-123] Scriptified get_recording_names and get_recording_list

### PR DESCRIPTION
**Pull request by JohanLaine**
Hello everyone!

### CODE REVIEW

File: get_recording_names.py [hunk 1]
```diff
@@ -12,37 +16,33 @@
 PREFIX = 'ABC-2727'
 
 
+def main():
+    if PREFIX == 'JIRA-1234':
+            sys.stderr.write('Please edit the file to set the PREFIX variable\n')
+            sys.exit(1)
```
Comment (hollste) [inline get_recording_names.py]: Please fix whitespace

File: get_recording_list.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 import time
 
+
```
Comment (hollste) [inline get_recording_list.py]: I liked the other whitespaces. Consider reverting b0ee1b958b9981fdd0a9f4105d8087e1c4bbbd87

File: get_recording_names.py (hunk 1 above)
Comment (rosmik) [reply get_recording_names.py]: Agree about fixing

File: get_recording_names.py [hunk 3]
```diff
@@ -29,16 +29,11 @@ def main():
...
-        ## Print as python list
-        print '['
-        for res in result_cursor:
-            print "'%s'," % res.name
-        print ']'
-    else:
-        ## Print one recording per line
-        for res in result_cursor:
-            print res.name
+## Print as python list
```
Comment (hollste) [inline get_recording_names.py]: Indendentation

File: get_recording_names.py [hunk 4]
```diff
@@ -2,39 +2,39 @@
...
 
+    if PREFIX:
+        query = {'name': {'$regex': '^' + PREFIX}}
+    else:
+        query = {}
+    fields = {'_id': False, 'name': True}
 
+    result_cursor = rapi.get_from_query(query, fields)
 
 ## Print as python list
```
Comment (hollste) [inline get_recording_names.py]: sad

File: get_recording_names.py [hunk 5]
```diff
@@ -2,39 +2,39 @@
...
 
-if PREFIX == 'JIRA-1234':
-	sys.stderr.write('Please edit the file to set the PREFIX variable\n')
-	sys.exit(1)
 
-cf = ConnectionFactory.production_connection()
-rapi = ApiFactory.recording_api()
+# Find all recordings with this prefix for example 'ABC-2481'
+# If PREFIX is an empty string, find all recordings
+PREFIX = 'ABC-2727'
```
Comment (hollste) [inline get_recording_names.py]: Please move this inside as well

File: get_recording_names.py [hunk 6]
```diff
@@ -2,39 +2,39 @@
...
 
 ## Print as python list
-print '['
-for res in result_cursor:
-    print "'%s'," % res.name
-print ']'
+    print '['
+    for res in result_cursor:
+        print "'%s'," % res.name
+    print ']'
```
Comment (rosmik) [inline get_recording_names.py]: bad style but ok

**Pull request merged**